chat/src/main/java/com/github/twitch4j/chat/util/ChatCrowdChant.java

```diff
@@ -0,0 +1,89 @@
+package com.github.twitch4j.chat.util;
+
+import com.github.twitch4j.chat.TwitchChat;
+import com.github.twitch4j.chat.events.channel.IRCMessageEvent;
+import com.github.twitch4j.common.annotation.Unofficial;
+import com.github.twitch4j.common.util.CryptoUtils;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Value;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import static com.github.twitch4j.chat.events.channel.IRCMessageEvent.NONCE_TAG_NAME;
+
+/**
+ * Information regarding a Crowd Chant participation or initiation.
+ *
+ * @see <a href="https://twitch.uservoice.com/forums/310201-chat/suggestions/43451310--test-crowd-chant">Related Uservoice</a>
+ */
+@Value
+@Unofficial
+public class ChatCrowdChant {
+
+    public static final String CHANT_MSG_ID_TAG_NAME = "crowd-chant-parent-msg-id";
+
+    /**
+     * The id for the parent message in the Crowd Chant.
+     */
+    String messageId;
+
+    /**
+     * The text being chanted.
+     */
+    String text;
+
+    /**
+     * Whether this message initiated a crowd chat.
+     * When false, the message is simply participating in the chant, rather than initiating.
+     */
+    boolean initiator;
+
+    /**
+     * The name of the channel where the Crowd Chant took place.
+     */
+    @Getter(AccessLevel.PRIVATE)
+    String channelName;
+
+    /**
+     * Sends the same message in the same channel to participate in the Crowd Chant, with the proper chat tag.
+     *
+     * @param chat an authenticated TwitchChat instance.
+     */
+    @Unofficial
+    public void participate(TwitchChat chat) {
```

Comment:
Needs to support `ITwitchChat` (so this will require another `sendMessage` overload in the interface so it can accept the tags map)